File: fase06/ex03/primo.js
function primo(x) {
    x = Math.abs(x)
if (x==2||x==3||x==5||x==7)
    return "Sim"
else if (x<=1||x!=Math.floor(x))
    return "Não"
else
for (i=0;i<(Math.floor(x/2));i++) {
    if ((x/(i+2))==Math.floor(x/(i+2)))
        return "Não"
    }
return "Sim"
}

File: python/poesiaVogon/fase06/ex03/primo.py
def primo(x) :
    import math
    if x==2 or x==3 or x==5 or x==7:
        return "Sim"
    elif x<=1 or x!=math.floor(x):
        return "Não"
    else:
        for i in range(1,x-1):
            if (x/(i+1))==math.floor(x/(i+1)):
                return "Não"
        return "Sim"

